src/lib.rs

```diff
@@ -199,10 +200,20 @@ pub mod words;
 
 #[doc(hidden)]
 #[inline(always)]
-pub fn unsafe_as_str(bytes: &[u8]) -> &str {
+pub const fn unsafe_as_str(bytes: &[u8]) -> &str {
 	// When used correctly by this crate's macros this should be safe
 	#[cfg(debug_assertions)]
-	return str::from_utf8(bytes).unwrap();
+	return match str::from_utf8(bytes) { Ok(s) => s, Err(_) => panic!("invalid str") };
```

Comment:
Just passing by, even if this is only internal: the `const` part should not be required in both cases anymore, right?